SU2_CFD/include/numerics_structure.hpp

```diff
@@ -2071,34 +2071,64 @@ class CUpwLin_AdjTurb : public CNumerics {
 };
 
 /*!
- * \class CUpwSca_TurbSA
- * \brief Class for doing a scalar upwind solver for the Spalar-Allmaral turbulence model equations.
+ * \class CUpwSca_Abstract
```

Comment:
No, I haven't looked into using the higher order upwinding in the turbulence classes.  For the most part, our group has had other priorities (like implementing a higher order time-stepping).